addons/analytic/models/analytic_plan.py

```diff
@@ -117,10 +117,11 @@ def action_view_analytical_accounts(self):
         return result
 
     def action_view_children_plans(self):
+        self.ensure_one()
```

Comment:
I hate the `ensure_one` mania.
There is already a `self.id` lower ensuring this, and I don't see why actions shouldn't generally allow to be run on multiple records if it can (no need to allow it here, just saying in general)